test/utils.test.js

```diff
@@ -101,3 +101,23 @@ QUnit.test('count of 0', function(assert) {
 QUnit.test('negative count', function(assert) {
   assert.deepEqual(range(-1), []);
 });
+
+QUnit.module('from');
+QUnit.test('simple array', function(assert) {
+  assert.deepEqual(from([1]), [1]);
+});
+
+QUnit.test('empty array', function(assert) {
+  assert.deepEqual(from([]), []);
+});
+
+QUnit.test('non-array', function(assert) {
+  assert.deepEqual(from(1), []);
+});
+
+QUnit.test('array-like', function(assert) {
+  var fixture = document.createElement('div');
+  fixture.innerHTML = '<div></div><div></div>';
+
+  assert.ok(from(fixture.getElementsByTagName('div')).map);
```

Comment:
Should this do some kind of check for the elements themselves (and length since no other test checked more than 1 element)?